app/controllers/concerns/hyku_addons/multitenant_locale_controller_behavior.rb

```diff
@@ -23,17 +23,16 @@ def default_url_options
       # See `I18n.fallbacks` for the registered fallback locales
       # @override
       def set_locale
-        I18nMultitenant.set(locale: requested_locale, tenant: current_tenant_name)
+        I18nMultitenant.set(locale: requested_locale, tenant: current_locale_name)
       end
 
       def requested_locale
         params[:locale] || I18n.default_locale
       end
 
-      # Use the setting if its been set or fall back to the tenants name.
-      # NOTE: The settings value can be "" or nil, so presence is used to check
-      def current_tenant_name
-        current_account&.settings&.dig("locale_name").presence || current_account&.name
+      # NOTE: The settings value can be "" or nil, so use `presence` if checking present?
+      def current_locale_name
+        current_account&.settings&.dig("locale_name")
```

Comment:
This changes it to opt-in via the setting instead of always on by default and overridable via the setting.  I could get behind this I think.  Make it explicit.

Reply:
My thinking was that there was no real need to have a custom locale set, then fall back to the default, if we are only ever going to trigger the custom locale when specific clients want to alter text around the admin. It seemed better to just use the default in that case.

Reply:
Looking at this again.  Do we need to restore the `.presence`?  Or does this still work if the setting is empty string `""`?